docs/release-notes/4420-substates-of-active

```diff
@@ -0,0 +1,11 @@
+:orphan:
+
+**API Changes**
+
+- The ``STATE_ACTIVE`` state for Experiments and Trials is divided into four sub-states:
+ ``STATE_QUEUED``, ``STATE_PULLING``, ``STATE_STARTING``, and ``STATE_RUNNING``.
+ Queries to ``GetExperimentsRequest`` which filter by state continue to use
```

Comment:
i think this points out a sort of interesting gap in the current implementation. Being able to filter by ACTIVE is still great, but would be a "nice to have" to allow filtering by the substates too. Maybe ticket it?

Reply:
Created https://determinedai.atlassian.net/browse/DET-8333